### ISSUE TITLE
Contributing a New Logo

### ISSUE THREAD

**Issue by artjeck** [mopidy/mopidy#1670]
Hi developers!
I am a graphic designer, I want to contribute to make a logo on your project, before I start it, I have to ask your permission first 😃
Regards, @artjeck

**Comment by jodal** [comment]
I'm in no way attached to our current "Mopidy ♪" "logo" in whatever font is currently used and would love to see some suggestions for a new logo.

The logo should have variations that work:

- As a favicon
- In a navbar, like on www.mopidy.com and discourse.mopidy.com
- On t-shirts? :-)

**Comment by artjeck** [comment]
![image](https://user-images.githubusercontent.com/38141262/39005282-0700ec04-442a-11e8-9aac-0f7d6ab0e104.png)

> Is this a currently used Mopidy logo Sir 

Can I see the original logo at this time?

**Comment by jodal** [comment]
Yes, this is the current logo. You can find it at https://github.com/mopidy/mopidy/blob/develop/docs/_static/mopidy.png

**Comment by artjeck** [comment]
oh yes sir 😉, means I can start making it right?
When should I finish it? 😁

**Comment by jodal** [comment]
Yes, anyone is free to and welcome to suggest a new project logo anytime!

As this is all based on voluntary contributions, when to start and end and what level of effort you put into the work is entirely up to you :-)

**Comment by artjeck** [comment]
Oh yeah sir, thanks for the permit. Once again thank you sir 😃

**Comment by artjeck** [comment]
Hello Sir 😁 
I have prepared a simple and elegant logos.
![background](https://user-images.githubusercontent.com/38141262/39072039-6957138a-4513-11e8-9b84-c67049c21270.jpg)

#### Navbar Display
![after1](https://user-images.githubusercontent.com/38141262/39072136-bcd892ae-4513-11e8-8d9d-142acb05dbdd.jpg)
![after2](https://user-images.githubusercontent.com/38141262/39072150-c84128fe-4513-11e8-9c3d-52573a646126.jpg)

#### Details Idea
![explain](https://user-images.githubusercontent.com/38141262/39072186-f2909a2c-4513-11e8-834c-d90a4ac4c2d4.jpg)

#### T Shirt Mock Up
![t-shirt mockup_front](https://user-images.githubusercontent.com/38141262/39072239-210ff08c-4514-11e8-8974-594abfc1f4ee.jpg)
![t-shirt mockup_back](https://user-images.githubusercontent.com/38141262/39072251-273b49b6-4514-11e8-9501-b96af71be72a.jpg)

#### Views on iOS Mobile
![mock up-1](https://user-images.githubusercontent.com/38141262/39072289-4bd423b0-4514-11e8-887e-1b4131db8892.jpg)
![mock up-2](https://user-images.githubusercontent.com/38141262/39072293-50d30746-4514-11e8-95ec-b55243e11214.jpg)

#### Logo Type
![navbar1](https://user-images.githubusercontent.com/38141262/39072371-8626f3bc-4514-11e8-975b-ec9c7e880f3d.png)
#### Logo Mark
for variations of the mark logo, I have prepared 2 versions, if you like other colors, Size, AI file type, SVG, PDF, PNG, JPG, no matter sir just say it, I hope you like it 😀 
How do you think sir?
![512_color](https://user-images.githubusercontent.com/38141262/39072403-9afaaf90-4514-11e8-9518-14e978e54e3f.png)
![512_transparant](https://user-images.githubusercontent.com/38141262/39072405-9b6a3d6a-4514-11e8-9db6-6d0ba779b9f8.png)

**Comment by artjeck** [comment]
Hi Owner 😀 
All File is [Here ](https://drive.google.com/drive/folders/1z44AaCG5qFrzMtQ4TEAXsmHq1XQdcCgM?usp=sharing)for new Logo Mopidy

**Comment by avanc** [comment]
Although the colour is not my favourite, I like the design.

**Comment by adamcik** [comment]
Reviving this since it never got that much traction. I would be happy to accept this contribution and switch logo. @jodal @kingosticks thoughts?

**Comment by kingosticks** [comment]
I do like the design and the thought behind it. I'm not sold on the colour either but it's not like Mopidy has any pre-existing colour branding. So all in all, a huge improvement. Thanks @artjeck!
